### PR TITLE
Migrate from Hugo to 11ty.

### PR DESCRIPTION
**Pull request by BrunoBernardino**
Very initial work, but already the previous asset URLs are preserved (like /uploads and such).

Related to #11

### CODE REVIEW

File: _includes/components/section-menu.njk [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
+            {% endif %}
 
-            {{ if not (or (.IsAncestor $currPage) (eq .Permalink $currPage.Permalink)) }}
-                {{ $hiddenAttr = "hidden" }}
-            {{ end }}
-
-            {{ if .IsNode }}
-                <ul {{ $hiddenAttr }}>
-                    {{ template "menu" (dict "page" . "currentPage" $currPage) }}
+            {% if (not page.anchors) or (page.anchors.length === 0) %}
```
Comment (BrunoBernardino) [inline _includes/components/section-menu.njk:28]: @MaKleSoft any idea why `page.anchors` doesn't even exist, here? I can't figure out why, and I'm hoping it's something simple/basic.

File: hilfe/handbuch/dienstplan/index.md [hunk 2]
```diff
@@ -1,43 +1,45 @@
 ---
+layout: layouts/hilfe/single.njk
 keywords: []
 title: Dienstplan
 weight: 1
 icon: table
+anchors: ["testing", "testing2"]
```
Comment (BrunoBernardino) [inline hilfe/handbuch/dienstplan/index.md]: Here's a simple example I was trying to pull, though these aren't "correct" anchors (they'd be a dictionary/object with `hash` and `title`, as per other articles).

File: _includes/components/section-menu.njk (hunk 1 above)
Comment (MaKleSoft) [reply _includes/components/section-menu.njk:28]: The table of contents is no longer provided explicitly by parsed automatically from the page content. You can reuse the logic from the Padloc page for that: https://github.com/padloc/site-new/blob/master/_lib/toc.js

Comment (MaKleSoft) [reply _includes/components/section-menu.njk:28]: Oh, and in 11ty, data from the front matter is available on the "top level". So the reference would simply be `anchors`, not `page.anchors`.

Comment (BrunoBernardino) [reply _includes/components/section-menu.njk:28]: Gotcha, but I'm inside a loop here, hence the `page`. It's coming from `eleventyNavigation` so maybe that's what's not providing the front matter data.

**Comment by BrunoBernardino** [comment]
@MaKleSoft this is ready for a review, the site has been migrated and has some noteworthy differences from the Hugo version:

- There's no CMS now (I've removed Forestry but haven't implemented a CMS yet, I was hoping that could be a separate PR - not sure if you've decided on anything yet -, but you let me know if that's not possible).
- There's a minor date format discrepancy on the articles. I kept the `toLocaleString()` (`August` vs `Aug`, and day before month), but I can tweak that if you want to look exactly the same everywhere.
- This site now has better SEO with a robots.txt file (didn't exist) and a sitemap (it existed before), with final file size optimizations (10% on average) and RSS/Atom/JSON feeds for the separate blog, news, and magazin sections.
- I still need to thoroughly go through and compare both sites to see if anything is different, but if you want to do that instead, be my guest, as I believe you'll have to check anyway.

I'm already way ahead on time this month, so I'll wait for your answer before continuing with this.

FYI @Lorenz147

**Comment by Lorenz147** [comment]
Good morning Bruno,

principally I would like you to stay within the monthly budget allocation. If the completion of a specific task/ project / job demands an extension, though, then it is so.
If it is possible for you, give me for budget reasons an estimated guess in advance.
In this case, let’s try to stay within the quarterly allocation.

Best
Lorenz

Von: Bruno Bernardino ***@***.***>
Gesendet: Donnerstag, 18. August 2022 18:42
An: pentacode/site ***@***.***>
Cc: Lorenz Strasser ***@***.***>; Mention ***@***.***>
Betreff: Re: [pentacode/site] Migrate from Hugo to 11ty. (PR #13)


@MaKleSoft<https://github.com/MaKleSoft> this is ready for a review, the site has been migrated and has some noteworthy differences from the Hugo version:

  *   There's no CMS now (I've removed Forestry but haven't implemented a CMS yet, I was hoping that could be a separate PR - not sure if you've decided on anything yet -, but you let me know if that's not possible).
  *   There's a minor date format discrepancy on the articles. I kept the toLocaleString() (August vs Aug, and day before month), but I can tweak that if you want to look exactly the same everywhere.
  *   This site now has better SEO with a robots.txt file (didn't exist) and a sitemap (it existed before), with final file size optimizations (10% on average) and RSS/Atom/JSON feeds for the separate blog, news, and magazin sections.
  *   I still need to thoroughly go through and compare both sites to see if anything is different, but if you want to do that instead, be my guest, as I believe you'll have to check anyway.

I'm already way ahead on time this month, so I'll wait for your answer before continuing with this.

FYI @Lorenz147<https://github.com/Lorenz147>

**Comment by BrunoBernardino** [comment]
Hi @Lorenz147 , I'll keep the monthly retainer the same, and any over time I do, it'll just carry over to the next month.

**Comment by Lorenz147** [comment]
We’ll do it the way you prefer.

Have a nice weekend!
Lorenz

Von: Bruno Bernardino ***@***.***>
Gesendet: Freitag, 19. August 2022 08:43
An: pentacode/site ***@***.***>
Cc: Lorenz Strasser ***@***.***>; Mention ***@***.***>
Betreff: Re: [pentacode/site] Migrate from Hugo to 11ty. (PR #13)


Hi @Lorenz147<https://github.com/Lorenz147> , I'll keep the monthly retainer the same, and any over time I do, it'll just carry over to the next month.

**Comment by BrunoBernardino** [comment]
@MaKleSoft just a reminder I'm blocked, waiting on a review on this (or just tell me the CMS needs to be implemented as part of this PR as well, so I can move forward with that, assuming you've chosen an option).

**Comment by MaKleSoft** [comment]
908 Files changed 😳😄

**Review by MaKleSoft — APPROVED** [review]
Ok, so I didn't go through the trouble of reviewing the actual code changes (for obvious reasons) but I ran it locally and everything looks pixel perfect! I'd say this is good to merge!

**Pull request merged**